hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMaster.java

```diff
@@ -2399,14 +2405,19 @@ private void startActiveMasterManager(int infoPort) throws KeeperException {
         Threads.sleep(timeout);
       }
     }
-    MonitoredTask status = TaskMonitor.get().createStatus("Master startup");
-    status.setDescription("Master startup");
+
+    // Here for the master startup process, we use TaskGroup to monitor the whole progress.
+    // The UI is similar to how Hadoop designed the startup page for the NameNode.
+    // See HBASE-21521 for more details.
+    boolean ignoreClearStartupStatus =
+      conf.getBoolean("hbase.master.ignore.clear.startup.status", true);
```

Comment:
im not really clear on the point of this config. looks like this boolean gets passed into the TaskGroup's createStatus call, but we never set that before. 

would prefer not to add new configs if possible. otherwise would like to be clearer on why one would ever want to change this value

Reply:
Good idea. Let the starup progress infomation be permanent in the MEM.